prolog/comment.pl

```diff
@@ -2,17 +2,36 @@
 module(tokenize(comment)
        [comment/2,
         comment_rec/2,
-        comment_token/2,
-        comment_token_rec/2]).
+        comment_token/3,
+        comment_token_rec/3]).
 */
-
-dcgtrue(U,U).
-
-id([X|L]) --> [X],id(L).
-id([]) --> dcgtrue.
-id([X|L],[X|LL]) --> [X],id(L,LL).
-id([],[]) --> dcgtrue.
-
+%% Author: Stefan Israelsson Tampe (stefan.itampe@gmail.com)
```

Comment:
Probably a good idea to format this as a module doc, following pldoc: http://www.swi-prolog.org/pldoc/man?section=sectioncomments